### ISSUE TITLE
Firmware update fails for Ledger Nano S: Stuck on Bootloader screen

### ISSUE THREAD

**Issue by devedse** [LedgerHQ/ledger-live-desktop#3312]
#### Ledger Live Version and Operating System

- tested on Ledger Live **Ledger Live 2.17.1 and 2.16.0**
- Platform and version: **Windows 10**

#### Expected behavior

I tried updating my Ledger Nano S after not using it for a very long time.

#### Actual behavior

When I tried updating everything seemed to go smoothly up until the point I needed to restart the ledger in Bootloader mode.

The bootloader screen shows on my Ledger but nothing happens on my PC. I just hear the sound of a USB device connecting but nothing happens.

![image](https://user-images.githubusercontent.com/2350015/100030576-215dd600-2df4-11eb-81e9-40d91a3511bc.png)

It'll just stay stuck at this screen.

#### Steps to reproduce the behavior

Do an update on my ledger device

**Comment by devedse** [comment]
I managed to workaround the issue by executing the update from my laptop. No idea why this works on that device and not on my computer.

**Comment by devedse** [comment]
Things got even weirder, I now tried to update my second ledger nano S on my laptop and ran into the same issue as I ran into with my computer. Somehow it just won't update.

**Comment by gre** [comment]
Do you happen to have other apps that could communicate to the device at the same time? It's a curious issue, maybe related to the USB stack / other devices plugged on the computer. I'm not sure otherwise.

**Comment by devedse** [comment]
@gre, nope nothing I know of.

**Comment by devedse** [comment]
Hmm I just started the application as administrator and then the update worked. I can't remember doing that the previous time though.

**Comment by gre** [comment]
ok 😢   I informed our firmware tech lead but the problem might be related to Windows's usb.
It might be related to the new firmware itself and something related to productID changes, but not 100% sure.

**Comment by greg7mdp** [comment]
I have the same problem. Tried the update on 3 computers and it always hang and I have the blinkling ledger icon on the nano S. Very frustrating!

**Comment by greg7mdp** [comment]
Even as administrator it doesn't work for me. My nano S is totally unusable. Is there any standalone program to reset it to factory settings?

**Comment by Komarac7** [comment]
I have the exactly same problem. After update USB recognizes only when I activate left button (bootloader) and after pressing continue on manager live it goes to 100% and the icon on ledger just keeps blinking. Tried multiple computers, multiple ports and multiple manager live versions and nothing helps. If i don't press the button the icon is standing still, and usb doesn't recognize it is connected. Right button shows "recovery" when plugged in, but also no sound from usb connection and nothing happens. Whith old version (1.4.1 I think) it worked and showed everything correctly. This happened after update to 1.6 version (not sure now was it 1.6.0 or 1.6.1 that was shown in manager)

**Comment by adamvp** [comment]
Try another usb cable

**Comment by Komarac7** [comment]
Tried from another dead ledger nano s, and it didn't help

**Comment by djspops** [comment]
New Nano S - will not go into setup mode; installed v2.20.exe for Win10;  Norton disabled;
Displays on "Welcome to Ledger Nano S", buttons do not work.
Help!

**Comment by rudedawg89** [comment]
I also am having the exact same issue. I emailed the Ledger Live help but have not received a response yet. If they contact me I'll let you all know what they say.

**Comment by maciejpawlak** [comment]
I had the same issue. Installing the Ledger Live on different device helped on bootloader step.

**Comment by midlan** [comment]
using administrator helped me

**Comment by rodneyratters** [comment]
I have same problem tried different leads usb ports run as admin , I had been reluctant to update in Google chrome bitcoin wallet app, as that worked fine back in 2019 , that was the last time i opened my Ledger nano s. Now trying Ledger live multiple hours night weeks.  Gets stuck on bootloader, admin did try & help & we seem to be making some success, until i went to check the still plugged in Nano s, all the work we had done was back on Nothing. to bootloader again. :( . Any ideas welcomed.

**Comment by shrtkv** [comment]
same here, stuck in boatloader after ledger interrupited the update due an error


website repair instructions doenst work

I was trying to update the firmware...the first update worked (1.3x to 1.6.0) them they asked another update, which crashed :(

**Comment by clashthe** [comment]
same

**Comment by asif0704** [comment]
I had same issue with my Nano X device. Device working fine but all of sudden its gone.

Currently, device unusable as stuck on Booting * % message. Booting took 8-10 hrs couple of times in the end no joy.

Tried changing USB etc. and follow instruction here in this thread already. Ran Ledger live on desktop as Administrator but same as left button not working on device.

Ledger support doesn't reply yet.

Ledger live version - 2.26.1.

Any help, much appreciated.

**Comment by flyingisawol** [comment]
same issue here. V 2.26.1
OSX 11.1

Am logged in as admin, tried changing usb cable.. cannot actually connect my device to ledger live app at all anymore it seems. Completely unresponsive.

Hoping for a fast resolution @ Ledger

**Comment by clashthe** [comment]
i solved by setup as new device on ledger hardware first

**Comment by rwperrott** [comment]
TLDR: An upgrade of my Ledger Nano S update, 1.40 to 1.6.0, never completes at 2nd of 3 dots.

- Installed ledger-live-desktop-2.39.2-linux-x86_64.AppImage on Linux Mint 20.3 MATE 64-bit.
- Started "Ledger Live".
- Connected the hardly used original USB cable directly to a USB 3 port (no hub), then plugged in my Ledger Nano S.
- It was recognised after PIN entry and the application flagged that an ungraded was needed from 1.40 to 1.60.
- Deleted all the apps as recommended.
- Started the upgrade.
- The progress circle got to 100% and I saw Update on the Ledger Nano S screen.
- It told me to disconnect and reconnect while holding the Left button to reboot, so I did, then left go of the button.
- The update never completed after several minutes.
- Stopped the update, then disconnected my Ledger Nano S.
- Restarted Ledger Live and retried with the same hold Left button down to cause bootloader start.
- It was recognised and I saw Update on the Ledger Nano S screen with "Preparing your device" on the Ledger Live, but this update never completes!

**Comment by rwperrott** [comment]
I got stuck at the 2nd dot of three, for 100% done there, on Linux Mint 20.3 MATE 64-bit, for a v1.40 to v1.60 upgrade for my Nano Ledger S.

I discovered repair mode, it showed a fix script, so I copying it, pasted it into a terminal windows and pressing enter. The repair then completed, and allowed me to further upgrade to, I think, v2.10.
